normandy/recipes/tests/test_api.py

```diff
@@ -212,12 +215,14 @@ def test_it_can_enable_recipes(self, api_client):
         recipe = Recipe.objects.all()[0]
         assert recipe.enabled
 
+    @pytest.mark.xfail(reason='disabled approvals temporarily')
     def test_cannot_enable_unapproved_recipes(self, api_client):
         recipe = RecipeFactory(approval=None, enabled=False)
 
         res = api_client.post('/api/v1/recipe/%s/enable/' % recipe.id)
         assert res.status_code == 400
 
+    @pytest.mark.skip(reason='disabled approvals temporarily')
```

Comment:
I don't think I'm happy with the idea of having unconditional skip tests. In sumo we had tests that were skipped in certain environments (like a developer without Elasticsearch). We made skips failures in CI. The only reason we don't do that in Normandy already is that I couldn't figure out how to do it.

I think we should either delete these tests, or leave them unmodified. They exist on the peer review branch. They don't provide any value here.